sequencer/src/api.rs

```diff
@@ -3643,7 +3649,7 @@ mod test {
         let coordinator = node_state.coordinator;
 
         let membership = coordinator.membership().read().await;
-        let block_reward = membership.block_reward();
+        let block_reward = membership.block_reward(None).expect("block reward is None");
```

Comment:
nit: for expect I think the idea is to put something that was expected, so in this case "not None"